kbase-extension/static/kbase/js/widgets/appWidgets2/input/checkboxInput.js

```diff
@@ -4,8 +4,8 @@ define([
     'common/events',
     'common/ui',
     'common/runtime',
-    '../validation',
-    '../validators/constants',
+    'widgets/appWidgets2/validation',
+    'widgets/appWidgets2/validators/constants',
```

Comment:
putting in the full path for these validators / validation modules so it's easier to find them in times of need